packages/firestore/test/util/helpers.ts

```diff
@@ -1073,3 +1075,44 @@ export function computeCombinations<T>(input: T[]): T[][] {
   };
   return computeNonEmptyCombinations(input).concat([[]]);
 }
+
+/**
+ * Helper method to generate bloom filter proto value for mocking watch
+ * existence filter response.
+ */
+export function generateBloomFilterProto(config: {
+  contains: MutableDocument[];
+  notContains: MutableDocument[];
+  hashCount: number;
+  numOfBits: number;
```

Comment:
nit: Please rename the `numOfBits` argument to `bitCount`, to match the property name in the BloomFilter class.